index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+</head>
+    <body>
+      <div class="home-page">
+        <header>
+
+            <nav class="hamburger">
+                <div class="log"><img src="img/My Logo.png" alt="Menu"></div>
+                <ul>
+                   
+                    <li class="log"><img src="img/Union.png" alt="Menu1"></li>
+            
+                </ul>
+            </nav>
+        </header>
```

Comment:
- [ ] You did great using the header and nav semantic tag here, :+1: But I think there are some changes you need to make here before moving to the next milestone.
- - The website logo is meant to be inside an `a` tag, which links to the home page. This is for three reasons: convention, semantics, and user experience, I'll suggest you update this accordingly. :+1: :100: 
- - Also looking at your design, there are two things that don't match the Figma design here.
- - - Your design has a lot of space before the menu and the logo which is not in the Figma design, I'll suggest you remove the too much space.
- - - Also the logo ad the hamburger menu is not perfectly aligned in a straight line, I'll suggest you update this too.

- - FIGMA DESIGN:
![Screenshot from 2023-06-06 15-48-20](https://github.com/medijdir/portfolio_project/assets/107632780/a041db79-9d84-4738-a6bf-7237ca91831e)

- - Your DESIGN:
![Selection_055](https://github.com/medijdir/portfolio_project/assets/107632780/af1cbf1d-4b13-44d6-856c-a16812e5f87b)